tests/telemetry/test_telemetry.py

```diff
@@ -240,3 +263,238 @@ def test_virtualdb_table_streaming(duthosts, rand_one_dut_hostname, ptfhost, loc
     assert_equal(len(re.findall('Max update count reached 3', result)), 1, "Streaming update count in:\n{0}".format(result))
     assert_equal(len(re.findall('name: "Ethernet0"\n', result)), 4, "Streaming updates for Ethernet0 in:\n{0}".format(result)) # 1 for request, 3 for response
     assert_equal(len(re.findall('timestamp: \d+', result)), 3, "Timestamp markers for each update message in:\n{0}".format(result))
+
+
+def run_gnmi_client(ptfhost, dut_ip):
+    """Runs python gNMI client in the corresponding PTF docker to query valid/invalid
+    tables in 'STATE_DB' on the DuT.
+
+    Args:
+        pfthost: PTF docker binding to the selected DuT.
+        dut_ip: Management IP of DuT.
+
+    Returns:
+        None.
+    """
+    gnmi_cli_cmd = 'python /root/gnxi/gnmi_cli_py/py_gnmicli.py -g -t {0} -p {1} -m subscribe --subscribe_mode 0\
```

Comment:
docker-ptf image will have different version for the location of gnxi. Could you please use fixture gnxi_path to avoid possible file location error?

Reply:
Thanks @ZhaohuiS! I saw you submitted a PR to fix the issue of `gnxi` path and will update this PR according to your suggestion.